### DISCUSSION TITLE
Clicking "View documentation" or "Feedback forum" opens 20 tabs

### DISCUSSION THREAD

**Discussion by nitsanb** [community/community#106639] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I just finished installing the plugin for IntelliJ and going through the Welcome Guide. The last page, "Learn More", has 2 links:
View documentation (https://docs.github.com/en/copilot)
Feedback forum (https://github.com/orgs/community/discussions/categories/copilot)

When I click either one, it opens that page in ~20 tabs (Google Chrome).
plugin 1.4.6.4092
IntelliJ IDEA 2022.3.3 (Community Edition)
MacOS 14.3

**Comment by nitsanb** [comment]
This doesn't happen with other links, like the chatbot's "waitlist" or "pre-release terms" links.

**Comment by Akash1134** [comment]
Hi @nitsanb ,

Thanks for reporting this issue. Does that happen if you change your default browser temporarily? I am not able to reproduce it -  if you could record a video illustrating the issue, that'll help us diagnose better. 

Thanks!

**Reply by nitsanb** [reply]
Yes it happens with Firefox as well.
I first reproduced it on Chrome again - this time with hundreds of tabs. But then I restarted IntelliJ - switching to Firefox and then back to Chrome it only opens two tabs.

https://github.com/community/community/assets/26364150/09eb7ee5-ad79-4f89-a4f7-ac99a90bb559


https://github.com/community/community/assets/26364150/1f32a9d7-591f-4b25-8760-5a50c57a4142

**Comment by dstekanov-tc** [comment]
I have faced the same issue (Mac, RubyMine) by clicking `View documentation` and `Feedback forum` at "Welcome to GitHub Copilot" tab.
